blocks/library/code/index.js

```diff
@@ -26,7 +26,7 @@ registerBlockType( 'core/code', {
 		return (
 			<TextareaAutosize
 				value={ attributes.content }
-				onFocus={ setFocus }
+				onFocus={ () => setFocus( {} ) }
```

Comment:
After 3f65f3c632646c2cee83639c6efc1455fea217bf, it doesn't even seem necessary anymore.